libs/clj-icat-direct/src/clj_icat_direct/queries.clj

```diff
@@ -310,13 +314,13 @@
     It returns the properly formatted SELECT query."
   [^String user ^String zone ^String parent-path ^String info-type-cond]
   (let [group-query "SELECT group_user_id FROM groups"]
-    (str "WITH groups    AS (" (mk-groups user zone) "),
-               objs      AS (" (mk-unique-objs-in-coll parent-path ) "),
-               file_avus AS (" (mk-obj-avus "SELECT data_id FROM objs") ")
-         " (mk-count-objs-of-type "objs" "file_avus" group-query info-type-cond))))
+    [[(mk-temp-table "groups" (mk-groups user zone))]
+     [(mk-temp-table "objs" (mk-unique-objs-in-coll parent-path))]
+     [(mk-temp-table "file_avus" (str "SELECT object_id, meta_attr_value, meta_attr_name FROM (" (mk-obj-avus "SELECT data_id FROM objs") ") obj_avus"))]
```

Comment:
I can't guarantee that I found all of the columns that were used, but it appears that the temporary table (or CTE, depending on the context) for AVUs is aliased to two names `f` and `m`. There were some cases where `m.meta_id` was referenced directly, but those were all cases where `m` was a direct alias to `r_meta_main` instead. I can't be absolutely certain at this point, but I believe that you can limit the fields selected by `mk-obj-avus`.